auth/client/src/test/resources/org/wildfly/security/auth/client/test-wildfly-config-openssh-keys.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2019, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<configuration>
+    <authentication-client xmlns="urn:elytron:client:1.4">
+
+        <credential-stores>
+            <credential-store name="store1">
+                <protection-parameter-credentials>
+                    <clear-password password="Elytron"/>
+                </protection-parameter-credentials>
+                <attributes>
+                    <attribute name="keyStoreType" value="JCEKS"/>
+                    <attribute name="create" value="true"/>
+                    <attribute name="location" value="target/ks-cred-stores/openssh-keys-test.jceks"/>
+                </attributes>
+            </credential-store>
+        </credential-stores>
+
+        <authentication-rules>
+
+            <rule use-configuration="test-rsa">
+                <match-host name="rsa"/>
+            </rule>
+
+            <rule use-configuration="test-dsa">
+                <match-host name="dsa"/>
+            </rule>
+
+            <rule use-configuration="test-ecdsa">
+                <match-host name="ecdsa"/>
+            </rule>
+
+            <rule use-configuration="test-ed25519-clear-pwd">
+                <match-host name="ed25519-clear-pwd"/>
+            </rule>
+
+            <rule use-configuration="test-ed25519-masked-pwd">
+                <match-host name="ed25519-masked-pwd"/>
+            </rule>
+
+            <rule use-configuration="test-ed25519-cred-store-ref">
+                <match-host name="ed25519-cred-store-ref"/>
+            </rule>
+
+        </authentication-rules>
+
+        <authentication-configurations>
+
+            <configuration name="test-rsa">
+                <credentials>
+                    <key-pair>
+                        <openssh-private-key openssh-private-key="-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Would be good to indicate how this openssh private key was generated / copied from (either here or in a separate readme file).

Reply:
Added readme file, thanks!